tests/installed/itest-bareuser-nouserxattrs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Test that initializing a bare-user repo on tmpfs fails
+# Maybe at some point this will be fixed in the kernel
+# but I doubt it'll be soon
+# https://www.spinics.net/lists/linux-mm/msg109775.html
+
+set -xeuo pipefail
+
+dn=$(dirname $0)
+. ${dn}/libinsttest.sh
+
+test_tmpdir=$(prepare_tmpdir)
+trap _tmpdir_cleanup EXIT
+cd ${test_tmpdir}
+
+mkdir mnt
+mount -t tmpfs tmpfs mnt
+if ostree --repo=mnt/repo init --mode=bare-user 2>err.txt; then
+    assert_not_reached "bare-user on tmpfs worked?"
+fi
+assert_file_has_content err.txt "Operation not supported"
+umount mnt
```

Comment:
The reason I didn't do that is they don't compose: https://stackoverflow.com/questions/3338030/multiple-bash-traps-for-the-same-signal

I pushed a fixup.